content/blog/2020/06/2020-07-07-custom-distribution-service.adoc

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 1 Summary"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+author: stellargo
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+
+Hello,
+
+=== Project Summary
+
+The main idea behind the project is to build a customizable jenkins distribution service that could be used to build tailor-made jenkins distributions. The service would provide users with a simple interface to select the configurations they want to build the instance with eg: plugins, authorization matrices etc. Furthermore it would include a section for sharing community created distros so that users can find and download already built jenkins war/configuration files to use out of the box.
+
+
+=== Quick review
+
+[width="25%",cols="20,^4"]
+|=======
+|Pull Requests Opened |48
+|Github Issues completed |60
+|=======
+
+=== Details
+
+I have written separate blog posts for every week in GSoC and the intricate details for each of them can be found at their respective blog pages. I am including a summary for every phase supported with the respective links. 
+
+==== Community Bonding
+
+This years gsoc had a longer community bonding than any of the previous editions and therfore this gave me a lot of time to explore, so I spent it by building a prototype for my project. I realised some of the blockages I might face early on, and therfore it gave me more clarity in terms of how I can proceed. I also spent this time preparing a design document which you can find link:https://docs.google.com/document/d/1-ujWVJ2a5VYkUF6UA7m4bEpSDxmb3mJZhCbmoKO716U/edit[here].
```

Comment:
Might be good to mention that could was the reason for the longer bonding period